src/stdatamodels/jwst/datamodels/wcs_ref_models.py

```diff
@@ -398,6 +399,107 @@ def to_fits(self):
         raise NotImplementedError("FITS format is not supported for this file.")
 
 
+class MiriLRSSpecwcsModel(ReferenceFileModel):
+    """
+    A model for a reference file of type "specwcs" for MIRI LRS Slit.
+    The model is for the specwcs for LRS Fixed Slit and LRSSlitless
```

Comment:
Do we expect an LRS slitless reference file?

Reply:
no we will not have an LRS slitless reference file

Reply:
@nden Let me clarify as understand it there is a reference file for FIXED slit and a reference file for SLITLESS. But Both of these files are read in with the same datamodel. The existing slitless reference on CRDS will be used.
A new reference file for FIXED SLIT LRS will have the v2,v3 vertices. 
Do you see any problem with this. I tested reading in the new reference file for LRS FIXED slit and the existing slitless reference file using this 1 datamodel model and they are read in correctly